### ISSUE TITLE
Set FPS limit from the Blender editor

### ISSUE THREAD

**Issue by moisesjpelaez** [armory3d/armory#3018]
After reading this in the [forums](https://forums.armory3d.org/t/set-limit-framerate-ln-for-get-fps/4286/9) and following this [issue](https://github.com/armory3d/iron/issues/107), I think the end user should be setting the FPS cap from `Output -> Frame Rate`:

![image](https://github.com/armory3d/iron/assets/24682046/a7e39a44-9b1f-40af-afda-0ddd18343cba)

**Comment by e2002e** [comment]
@moisesjpelaez you cannot set the framerate in armory3D. It will run at default speed normalized by a timer at the maximum speed it can.

**Comment by moisesjpelaez** [comment]
> @moisesjpelaez you cannot set the framerate in armory3D. It will run at default speed normalized by a timer at the maximum speed it can.

Yeah I realized it, I've seen Kha needs to be modified (as far as I'm understanding) and this isn't ideal at all for end users that want to make games in Armory. There's also [an issue](https://github.com/armory3d/armory/issues/3014) related to the default maximum speed with Krom/Windows builds (my games run twice as fast in my 120Hz monitor), the proper speed only works as it should with HTML5 builds.
